services/openvpn/client.go

```diff
@@ -62,6 +64,11 @@ func (c *Client) Start(options connection.ConnectOptions) error {
 	}
 	c.process = proc
 	log.Infof("client config: %v", clientConfig)
+	removeAllowedIPRule, err := firewall.AllowIPAccess(clientConfig.VpnConfig.RemoteIP)
```

Comment:
```suggestion
	c.removeAllowedIPRule, err := firewall.AllowIPAccess(clientConfig.VpnConfig.RemoteIP)
```
direct assignment to struct member should be sufficient here..

Reply:
Nope :) if error is returned other return values are undefined according to go. So we can asign removal function only if error was nil.